lib/ceedling/file_path_utils.rb

```diff
@@ -21,9 +21,11 @@ class FilePathUtils
 
   # standardize path to use '/' path separator & have no trailing path separator
   def self.standardize(path)
-    path.strip!
```

Comment:
The `html_absolute_paths` option caused an exception when project.yml was parsed because the option is boolean. I changed this function to replace \ with / only if the path is a string.